app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def bulma_flash_mapping(message_type)
+    case message_type
+    when 'alert'
```

Comment:
yes! let's follow the rails ways as closer as possible with doing mapping to other frameworks on the interface layer.